specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -6888,6 +7358,276 @@
     }
   },
   "definitions": {
+    "RestorePointCollectionProperties": {
+      "properties": {
+        "source": {
+          "$ref": "#/definitions/SubResource",
+          "description": "Reference Id of the VM for which the restore points have to be created."
+        },
+        "restorePoints": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RestorePoint"
+          },
+          "description": "Gets a list containing all restore points created under this restore point collection."
+        }
+      },
+      "description": "The restore point collection properties."
+    },
+    "RestorePointCollection": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RestorePointCollectionProperties"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Create or update Restore Point collection parameters."
+    },
+    "RestorePointCollectionUpdate": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RestorePointCollectionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/UpdateResource"
+        }
+      ],
+      "description": "Update Restore Point collection parameters."
+    },
+    "RestorePointCollectionListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RestorePointCollection"
+          },
+          "description": "Gets the list of restore point collections."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The uri to fetch the next page of OS Upgrade History. Call ListNext() with this to fetch the next page of history of upgrades."
+        }
+      },
+      "description": "The List restore point collection operation response."
+    },
+    "RestorePointSourceMetadata": {
+      "properties": {
```

Comment:
Need to add the reference to SecurityProfile

Reply:
added it